### PR TITLE
Add logrus.WithFields for all V2 calls

### PR DESCRIPTION
**Pull request by jterry75**
Signed-off-by: Justin Terry (VM) <juterry@microsoft.com>

### CODE REVIEW

File: service/gcs/core/gcs/storage.go [hunk 1]
```diff
@@ -185,14 +190,14 @@ func scsiControllerLunToName(osl oslayer.OS, controller, lun uint8) (string, err
 
 // scsiLunToName finds the SCSI device with the given LUN. This assumes
 // only one SCSI controller.
-func scsiLunToName(osl oslayer.OS, lun uint8) (string, error) {
-	return scsiControllerLunToName(osl, 0, lun)
+func (c *gcsCore) scsiLunToName(lun uint8) (string, error) {
```
Comment (jterry75) [inline service/gcs/core/gcs/storage.go]: This was V1 only. Marking that explicitly here. If we want this a separate change I can do that.

File: service/gcs/core/gcs/storage.go [hunk 2]
```diff
@@ -185,14 +190,14 @@ func scsiControllerLunToName(osl oslayer.OS, controller, lun uint8) (string, err
...
+	return scsiControllerLunToName(c.OS, 0, lun)
 }
 
 // deviceIDToName converts a device ID (scsi:<lun> or pmem:<device#> to a
 // device name (/dev/sd? or /dev/pmem?).
 // For temporary compatibility, this also accepts just <lun> for SCSI devices.
-func deviceIDToName(osl oslayer.OS, id string) (device string, pmem bool, err error) {
+func (c *gcsCore) deviceIDToName(id string) (device string, pmem bool, err error) {
```
Comment (jterry75) [inline service/gcs/core/gcs/storage.go]: This was V1 only. Marking that explicitly here. If we want this a separate change I can do that.

File: service/gcsutils/fs/ext4.go [hunk 3]
```diff
@@ -134,32 +134,25 @@ func (e *Ext4Fs) CleanupSizeContext() error {
...
-	if err != nil {
-		logrus.Infof("running mkfs.ext4 failed with ... (%s)", err)
-	}
-
-	return err
-}
-
-// MakeBasicFileSystem just creates an empty file system on the given file using
-// the default settings.
-func (e *Ext4Fs) MakeBasicFileSystem(file *os.File) error {
```
Comment (jterry75) [inline service/gcsutils/fs/ext4.go]: Unused code.

**Comment by jterry75** [comment]
I have verified all this output and logging. Looks good for now.

File: service/gcs/bridge/bridge.go [hunk 4]
```diff
@@ -621,12 +668,19 @@ func (b *Bridge) signalContainer(w ResponseWriter, r *Request, signal oslayer.Si
 		return
 	}
 
+	fields := logrus.Fields{
```
[the remark this answers is not in the document]
Comment (jterry75) [reply service/gcs/bridge/bridge.go]: Fair enough.

File: service/gcs/core/gcs/uvm.go [hunk 5]
```diff
@@ -466,21 +490,27 @@ func (c *Container) AddNetworkAdapter(o oslayer.OS, a *prot.NetworkAdapterV2) er
 		}
 		break
 	}
-	logrus.Debugf("Waited: %v for the network adapter to show up", time.Now().Sub(start))
+	log.Data["adapter-wait-time-ns"] = time.Now().Sub(start).Nanoseconds
```
Comment (kevpar) [inline service/gcs/core/gcs/uvm.go]: Could use time.Since(start) if you prefer

**Pull request merged**